packages/myst-to-jats/src/index.ts

```diff
@@ -127,7 +136,9 @@ const handlers: Record<string, Handler> = {
     state.renderChildren(node);
   },
   blockquote(node, state) {
+    state.openNode('p');
     state.renderInline(node, 'disp-quote');
+    state.closeNode();
```

Comment:
Is this necessary?

![image](https://github.com/executablebooks/mystjs/assets/913249/ee9d8607-de4c-4243-aaf3-ca5cdd6136c7)

Reply:
I added this when testing a quote inside a figure caption, which isn't allowed. That's too much of an edge case though, so I reverted this change.